src/components/Preview/Preview.tsx

```diff
@@ -131,7 +133,13 @@ export const Preview = ({
   }, [error.title]);
 
   useEffect(() => {
-    hmsActions.preview(config);
+    try {
+      hmsActions.preview(config).catch((e: any) => {
```

Comment:
.catch should be sufficient I think. try catch is not necessary. Also can we make the log more descriptive

Reply:
Accidental commit - reverted now. Preview needs a better error handling. I'll commit this as a part of a separate PR. Please ignore this for now.